test/fixture/generatedTypeScript/interfaces/IExtension.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable import/prefer-default-export, import/no-cycle, @typescript-eslint/naming-convention, @typescript-eslint/no-empty-interface */
+import { 
+  ICoding,
+  IElement,
+  IPrimitiveBase64Binary,
+  IPrimitiveBoolean,
+  IPrimitiveCanonical,
+  IPrimitiveCode,
+  IPrimitiveDate,
+  IPrimitiveDateTime,
+  IPrimitiveDecimal,
+  IPrimitiveId,
+  IPrimitiveInstant,
+  IPrimitiveInteger,
+  IPrimitiveMarkdown,
+  IPrimitiveOid,
+  IPrimitivePositiveInt,
+  IPrimitiveString,
+  IPrimitiveTime,
+  IPrimitiveUnsignedInt,
+  IPrimitiveUri,
+  IPrimitiveUrl,
+  IPrimitiveUuid,
+} from "../internal";
+
+export interface IExtension extends IElement {
```

Comment:
Codacy found an issue: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)